pkg/storage/db.go

```diff
@@ -117,16 +117,11 @@ func (db *DB) RunScript(ctx context.Context, name string) error {
 }
 
 // Init initializes a connection to a Postgres database according to the environment variables POSTGRES_HOST, POSTGRES_PORT, POSTGRES_USER, POSTGRES_PASSWORD, POSTGRES_DATABASE
-func (db *DB) Init(ctx context.Context, postgresConfig *PostgresConfig) error {
+func (db *DB) Init(ctx context.Context, host string, port string, user string, password string, dbname string, partitionTTL int) error {
```

Comment:
nit:

This is borderline crossing into too many parameter territory at 6 but I see it reduces the the redundant variables we must set below.

I like it in this case but if we need to add more variables to this `Init` in the future then we might consider going back to the struct.